src/config.rs

```diff
@@ -5,15 +5,17 @@ use getset::{CopyGetters, Getters};
 use log::LevelFilter;
 use serde::{Deserialize, Serialize};
 use std::path::PathBuf;
+use strum::EnumString;
 
 #[derive(Builder, Clap, CopyGetters, Getters, Deserialize, Serialize)]
 #[builder(default, pattern = "owned", setter(into))]
 #[serde(rename_all = "kebab-case")]
 #[clap(
+    about("CRI - The Kubernetes Container Runtime written in Rust"),
```

Comment:
as an example. this is not really part of this PR... I think this is fine in the current state of this repo (a very young hacking project). Just wanted to get your all thoughts/input so I can move forward and make my reviews better for you all.

Reply:
Yeah, I agree. I admit being too lazy to open another PR for this change :innocent: 

The main reason to change this was to fix the comment in `cargo doc` while still preserving the line on the CLI output.